sui_types/src/object.rs

```diff
@@ -400,6 +400,20 @@ impl Object {
         ObjectDigest::new(sha3_hash(self))
     }
 
+    // Size of the object in bytes.
+    // TODO: For now we just look at the size of the data.
+    // Do we need to be accurate and look at the serialized size?
```

Comment:
I think the way you are currently doing this is safe. But of course we have to be careful not to do it in a way that relies on how Rust represents things in-memory.